### PR TITLE
Inline doc for Series APIs

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR closes #236 to resolve conflicts and doc corrections.

### CODE REVIEW

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -316,8 +316,65 @@ def index(self):
...
+        --------
+        >>> s = ks.Series([1, 2, 3, 4], name='foo',
+        ...         index=pd.Index(['a', 'b', 'c', 'd'], name='idx'))
+        >>> s.reset_index()
+          idx  foo
+        0   a    1
+        1   b    2
+        2   c    3
+        3   d    4
+        >>> s.reset_index(name='values')
```
Comment (rxin) [inline databricks/koalas/series.py]: need a new line separating different examples

File: databricks/koalas/series.py [hunk 2]
```diff
@@ -468,8 +525,33 @@ def to_pandas(self):
...
+        >>> df = ks.DataFrame({'age': [5, 6, np.NaN],
+        ...    'born': [pd.NaT, pd.Timestamp('1939-05-27'),
+        ...             pd.Timestamp('1940-04-25')],
+        ...    'name': ['Alfred', 'Batman', ''],
+        ...    'toy': [None, 'Batmobile', 'Joker']})
+        >>> df.age.isnull() # doctest: +NORMALIZE_WHITESPACE
+        0    False
+        1    False
+        2     True
+        Name: ((age IS NULL) OR isnan(age)), dtype: bool
```
Comment (rxin) [inline databricks/koalas/series.py]: this name is probably different from what pandas does. similar issue to https://github.com/databricks/koalas/pull/273#discussion_r282307905 ?

File: databricks/koalas/series.py [hunk 3]
```diff
@@ -478,23 +560,113 @@ def isnull(self):
...
         else:
             return ks
 
-    @derived_from(DataFrame)
     def head(self, n=5):
+        """
+        Return the first n rows.
+        This function returns the first n rows for the object based on position.
+        It is useful for quickly testing if your object has the right type of data in it.
+        Parameters
```
Comment (rxin) [inline databricks/koalas/series.py]: i feel like the blank lines are lost

File: databricks/koalas/series.py (hunk 2 above)
Comment (HyukjinKwon) [reply databricks/koalas/series.py]: Yes, I think so.

File: databricks/koalas/series.py [hunk 4]
```diff
@@ -478,23 +574,131 @@ def isnull(self):
...
+        >>> ser = ks.Series([np.NaN, 2, pd.NaT, '', None, 'I stay'])
+        >>> ser
+        0       NaN
+        1         2
+        2       NaT
+        3
+        4      None
+        5    I stay
+        dtype: object
+        >>> ser.dropna()
```
Comment (rxin) [inline databricks/koalas/series.py]: new line?

File: databricks/koalas/series.py [hunk 5]
```diff
@@ -478,23 +574,131 @@ def isnull(self):
...
+        Detect existing (non-missing) values.
+        Return a boolean same-sized object indicating if the values are not NA.
+        Non-missing values get mapped to True.
+        Characters such as empty strings '' or numpy.inf are not considered NA values
+        (unless you set pandas.options.mode.use_inf_as_na = True).
+        NA values, such as None or numpy.NaN, get mapped to False values.
+
+        Returns
+        -------
+        Series: Mask of bool values for each element in Series
```
Comment (rxin) [inline databricks/koalas/series.py]: a space before : ?

**Comment by rxin** [comment]
lgtm

**Pull request merged**